index.html

```diff
@@ -4,12 +4,33 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Hello Microverse</title>
+    <title>Setup Mobil Version</title>
     <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@700&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500&display=swap" rel="stylesheet">
 </head>
 <body>
-    <main>
-        <h1 class="greeting">Hello Microverse!</h1>
-    </main>
+  <header>
+    <div class="header__status">
+      <span><img src="./images/time.png" alt="time"></span>
+      <span><img src="./images/status-bar.png" alt="status bar"></span>  
+    </div>
```

Comment:
- [ ] I think it would be better if you remove these code lines as they are not part of the project. They appear because of the device used to take a screenshot.